lib/actions/ssh.ts

```diff
@@ -80,30 +79,29 @@ async function getContainerId(
 		// container
 		const { child_process } = await import('mz');
 		const escapeRegex = await import('lodash/escapeRegExp');
-		const { getSubShellCommand } = await import('../utils/helpers');
+		const { which } = await import('../utils/helpers');
 		const { deviceContainerEngineBinary } = await import('../utils/device/ssh');
 
-		const command = generateVpnSshCommand({
+		const sshBinary = await which('ssh');
```

Comment:
This translates `'ssh'` to something like:
* `/usr/bin/ssh` (macOS)
* `C:\WINDOWS\System32\OpenSSH\ssh.EXE` (Powershell, cmd.exe)
* `C:\msys64\usr\bin\ssh.EXE` (MSYS2)

... without relying on invoking a shell to do it, instead looking directly at the PATH env var (which is what the shells do), thus allowing us to avoid dealing with the quoting and escaping peculiarities of each shell / platform combination.